There is a test failure with SciPy main:
```python
_____________ sympy/stats/tests/test_joint_rv.py:test_sample_scipy _____________
Traceback (most recent call last):
  File "/home/runner/work/sympy/sympy/sympy/stats/tests/test_joint_rv.py", line 338, in test_sample_scipy
    samps = sample(X, size=size)
            ^^^^^^^^^^^^^^^^^^^^
  File "/home/runner/work/sympy/sympy/sympy/stats/rv.py", line 1182, in sample
    return next(iterator)
           ^^^^^^^^^^^^^^
  File "/home/runner/work/sympy/sympy/sympy/stats/rv.py", line 1352, in return_generator_finite
    d = ps.sample(size=(numsamples,) + ((size,) if isinstance(size, int) else size),
        ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/runner/work/sympy/sympy/sympy/stats/joint_rv.py", line 143, in sample
    return {RandomSymbol(self.symbol, self): self.distribution.sample(size,
                                             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/runner/work/sympy/sympy/sympy/stats/joint_rv.py", line 323, in sample
    samps = _get_sample_class_jrv[library](self, size, seed=seed)
            ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/runner/work/sympy/sympy/sympy/stats/joint_rv.py", line 153, in __new__
    return cls._sample_scipy(dist, size, seed)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/runner/work/sympy/sympy/sympy/stats/joint_rv.py", line 186, in _sample_scipy
    samples = scipy_rv_map[dist.__class__.__name__](dist, size)
              ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/runner/work/sympy/sympy/sympy/stats/joint_rv.py", line 169, in <lambda>
    'MultivariateBetaDistribution': lambda dist, size: scipy_stats.dirichlet.rvs(
                                                       ^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/opt/hostedtoolcache/Python/3.11.0-beta.1/x64/lib/python3.11/site-packages/numpy/lib/utils.py", line 95, in newfunc
    warnings.warn(depdoc, DeprecationWarning, stacklevel=2)
    ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
DeprecationWarning:         `dirichlet.rvs` is deprecated, use `multivariate_beta.rvs` instead!
          # noqa
        `dirichlet` is deprecated due to an interface inconsistency: compared to
        other distributions, methods `pdf` and `logpdf` expect the transpose of the
        input `x`. Please use `multivariate_beta`, which corrects this inconsistency.
        In SciPy 1.11.0, this deprecation warning will be removed and `dirichlet` will
        become an alias for `multivariate_beta`.
```
I mentioned it upstream here: https://github.com/scipy/scipy/pull/16042#issuecomment-1137227051

I don't know if SciPy might change that because it doesn't look like there is any reason to deprecate the `dirichlet.rvs` method.

--- a/.github/workflows/runtests.yml
+++ b/.github/workflows/runtests.yml
@@ -166,7 +166,7 @@ jobs:
           python-version: ${{ matrix.python-version }}
 
       # Install the non-Python dependencies
-      - run: sudo apt install antlr4 libgfortran5 gfortran libmpfr-dev libmpc-dev
+      - run: sudo apt install antlr4 libgfortran5 gfortran libmpfr-dev libmpc-dev libopenblas-dev
       - run: python -m pip install --upgrade pip wheel
 
       # Use numpy and scipy from git for Python 3.11

